app/Http/Controllers/IndexController.php

```diff
@@ -28,16 +28,17 @@ public function __construct(ConfigFunctions $configFunctions)
 
 	public function show()
 	{
+		Configs::get();
 
 		if (Configs::get_value('landing_page_enable', '0') == '1')
 		{
-			$lang = Lang::get_lang(Configs::where('key', '=', 'lang')->first());
+			$lang = Lang::get_lang(Configs::get_value('lang', 'en'));
 
 			$infos = $this->configFunctions->get_pages_infos();
 
 			$menus = Page::menu()->get();
 
-			$title = Configs::get_value('site_title');
+			$title = Configs::get_value('site_title', 'Lychee v4');
```

Comment:
Given that `Lychee v4` is repeated a bunch of times, wouldn't it make sense to define it in one of the `config/` files?

Reply:
good point.